src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
The main issue is that the Srgb primaries definition is incorrect.

I was thinking of this
```{rust}
pub trait RgbStandard {
    type Primaries: Primaries;
    type WhitePoint: WhitePoint;
    type TransferFn: TransferFn;
}
```
The white point of the primaries have no relation to the color space white point. The primaries trait needs its own white point 
```{rust}
pub trait Primaries {
    type Wp: Whitepoint

    fn red< T: Float>() -> Yxy<Wp, T>;
    fn green<T: Float>() -> Yxy<Wp, T>;
    fn blue< T: Float>() -> Yxy<Wp, T>;
}
```

Srgb primaries will be implemented as

```{rust}
impl Primaries for Srgb {
   type Wp = D65;
    fn red< T: Float>() -> Yxy<Wp, T> {
      Yxy::with_wp(flt(0.6400), flt(0.3300), flt(0.212656));
    }
    ...
```

We could also implement the white point as follows, but it will make the implementation a bit cumbersome. 

I'd rather have a simple definition of the white point and let the library handle the adaptations where necessary
```{rust}
impl Primaries for Srgb {
   type Wp = D65;
    fn red< T: Float>() -> Yxy<Wp, T> {
      let yxy: Yxy<D60, T> = Yxy::with_wp(flt(0.6400), flt(0.3300), flt(0.212656));
      if Wp == D65 {
        yxy
      } else {
        yxy.adapt_into()
      }
    }
    ...
```